travis/test_pylint

```diff
@@ -136,9 +136,13 @@ pylint_rcfile = os.path.join(
     'cfg',
     "travis_run_pylint.cfg")
 modules_cmd = []
-if os.environ.get('INCLUDE_LINT'):
-    for path in os.environ.get('INCLUDE_LINT').split(' '):
-        modules_cmd.extend(['--path', path])
+if os.environ.get('INCLUDE_LINT') or os.environ.get('EXCLUDE_LINT'):
+    if os.environ.get('INCLUDE_LINT'):
```

Comment:
In fact, we could avoid the `if` using:
`for path in (os.environ.get('INCLUDE_LINT') or '').split(' ')`